docs/release-info/release-notes.md

```diff
@@ -333,6 +336,7 @@ Added the following fields from the query-based ingestion task report to the res
 - Improved query filtering to correctly process cases where both an IN expression and an equality (`=`) filter are applied to the same string value [#16597](https://github.com/apache/druid/pull/16597)
 - Improved the speed of SQL IN queries that use the SCALAR_IN_ARRAY function [#16388](https://github.com/apache/druid/pull/16388)
 - Improved the ARRAY_TO_MV function to handle cases where an object selector encounters a multi-value string [#17162](https://github.com/apache/druid/pull/17162)
+- Improved query filtering so that Druid tries to arrange query filters based on the computational cost of bitmap indexes, prioritizing less expensive filters for computation first. Filters with high compute costs relative to the number of rows they can filter might be omitted. [#17055](https://github.com/apache/druid/pull/17055)
```

Comment:
#17125 is the PR to compute the cost (following #17055)

Reply:
```suggestion
- Improved query filtering so that Druid tries to arrange query filters based on the computational cost of bitmap indexes, prioritizing less expensive filters for computation first. Filters with high compute costs relative to the number of rows they can filter might be omitted. [#17055](https://github.com/apache/druid/pull/17055) [#17125](https://github.com/apache/druid/pull/17125)
```